Makefile

```diff
@@ -11,11 +11,14 @@ PKCS11_ENGINE_PATH ?= "/usr/lib/x86_64-linux-gnu/engines-3/pkcs11.so"
 
 
 TASKS = config build format tidy garage-tools custom-client
-.PHONY: $(TASKS) test
+.PHONY: $(TASKS) test test-uptane-vectors
```

Comment:
I wonder if `all $(TASKS)` would work for if `test-uptane-vectors` is added to the list of `TASKS`?